src/Runtime/ServiceManager.php

```diff
@@ -12,6 +12,7 @@
 use Consolidation\SiteAlias\SiteAliasManagerAwareInterface;
 use Consolidation\SiteProcess\ProcessManagerAwareInterface;
 use Drupal\Component\DependencyInjection\ContainerInterface as DrupalContainer;
+use Drupal\Core\Recipe\RecipeCommand;
```

Comment:
would be good to add some readme or doc somewhere on how to run this, however simple it may seem

Reply:
the recipe command is documented like any other command - `drush list` to see its name and then `drush recipe --help`. It also has a docs page - https://www.drush.org/13.x/commands/recipe/. 

do you have anything else in mind?

Reply:
ah, got it, thanks Moshe